### PR TITLE
Added the full_name attribute to the dictionary and to the User object.

### PR DESCRIPTION
**Pull request by pagbabian-splunk**
As per Issue #649

### CODE REVIEW

File: dictionary.json [hunk 1]
```diff
@@ -1408,6 +1408,11 @@
       "description": "The email header From values, as defined by RFC 5322.",
       "type": "email_t"
     },
+    "full_name": {
+      "caption": "Full Name",
+      "description": "The full name of the person, as per the LDAP commonName attribute.",
```
Comment (rroupski) [inline dictionary.json:1413]: Perhaps replace "LDAP commonName" with "LDAP Common Name (CN) -- using the LDAP user-friendly name"?

Comment (pagbabian-splunk) [reply dictionary.json:1413]: the actual attribute in LDAP is commonName.  I said attribute, which is `commonName` but I can drop that to say LDAP Common Name (CN).

Comment (pagbabian-splunk) [reply dictionary.json]: It's been updated in Files.

**Review by Aniak5 — APPROVED** [review]
LGTM

**Review by rroupski — APPROVED** [review]

**Pull request merged**